lib/llvmopencl/LLVMFileUtils.cc

```diff
@@ -240,44 +233,48 @@ int pocl_mk_tempname(char *output, const char *prefix, const char *suffix,
 
 /****************************************************************************/
 
+#define CHUNK_SIZE (2 * 1024 * 1024)
+
 int
 pocl_read_file(const char* path, char** content, uint64_t *filesize) {
     assert(content);
     assert(path);
     assert(filesize);
+    *content = nullptr;
+    *filesize = 0;
 
     int fd;
     std::error_code ec;
     Twine p(path);
 
-    *content = NULL;
-
-    int errcode = pocl_filesize(path, filesize);
-    if (errcode)
-      return errcode;
-
-    size_t fsize = (size_t)(*filesize);
+    /* files in /proc return zero size, while
```

Comment:
Did you run the new clang formatter? These comments seem to not fill the line.

Reply:
Yes, i did. It doesn't do anything on that commit.